### PR TITLE
Update to discord-player v6

### PR DESCRIPTION
**Pull request by iGameCreep**
This pull request has been opened to update the bot to the discord-player v6.0.0 version as it is the newest & contains less bugs ( like the song stopping randomly ).
This code should be working perfectly, except for the seek command, because it isn't working normally : it comes from the discord-player package but should be fix in a newer version of the package.

Updated commands, events & main file to work with the new methods of the package.

If you find any bugs, don't hesitate to comment on this PR / create an issue on my repository.

### CODE REVIEW

**Comment by BabaYagaPROP** [comment]
Bot leaves the same second it joins the channel. It says " was manually disconnected from the voice channel, clearing queue...".

**Comment by iGameCreep** [comment]
> Bot leaves the same second it joins the channel. It says " was manually disconnected from the voice channel, clearing queue...".

Hello ! I'll look through this issue.

**Comment by iGameCreep** [comment]
I am not having this issue, does anything shows up on the console ? If you prefer to continue this issue on Discord, you can open a post in #support ( https://discord.gg/zerio-s-playground-773639842449850368 ). If not, it can be continued here.

**Comment by sterrebloem** [comment]
Hi, I have a question regarding the /play command. I used your code to update to discord-player v6 and for most part, that fortunately went well. However, I encountered a weird glitch. When I add new songs to the queue, the bot immediately skips to the next song in the queue (doesn't have to be the song you just linked). Same happens with the /search command. 

Is there anything I could be possibly doing wrong? Or is this something you encountered before? Many thanks in advance for your answer :)

**Comment by iGameCreep** [comment]
Bugs with play and test commands should be fixed ! I guess I missed them while updating methods.
Thanks again for letting me know !

File: commands/music/search.js [hunk 1]
```diff
@@ -72,7 +72,7 @@ module.exports = {
 
             queue.addTrack(res.tracks[query.content - 1]);
 
-            if (!queue.playing) await queue.node.play();
+            if (!queue.isPlaying) await queue.node.play();
```
Comment (skdhg) [inline commands/music/search.js:75]: isPlaying needs ()

**Comment by iGameCreep** [comment]
Typo, my bad 🤦
I'll fix this quickly.

File: commands/music/search.js (hunk 1 above)
Comment (iGameCreep) [reply commands/music/search.js:75]: Typo, my bad 🤦
I'll fix this quickly.

**Comment by iGameCreep** [comment]
Updated code to work with discord-player v6.1.1:
- Fix of a few bugs
- Fiw of the seek command

**Comment by revot334** [comment]
updated in v8.0.0... update will be out in a few days!

**Pull request closed without merging**